<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DMDController</class>
 <widget class="QMainWindow" name="DMDController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DMDController</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="fImage0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>251</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="fImage1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>251</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="leImage0Filename">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbImage0GetFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>107</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Image</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leImage1Filename">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbImage1GetFile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>107</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Image</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSwapImages">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Image Swap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbUploadImages">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>107</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPauseSequencer">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>106</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRunSequencer">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>106</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbShutDown">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Shut Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTest">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbResult">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbHScrollPlus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbHScrollMinus">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leHScroll">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbVScrollMinus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leVScroll">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbVScrollPlus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbHorShiftLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Hor. Shift:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbVerShiffLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver. Shift:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbHorShiftValue">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbVerShiftValue">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMapFilename">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMapGetFile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>107</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Map</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbMapImages">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Image Map</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSeqFilename">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSeqGetFile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>107</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Sequence</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
